types/conf.go

```diff
@@ -330,3 +330,42 @@ func CheckSystemNamespaces(namespace string, systemNamespaces []string) bool {
 	}
 	return false
 }
+
+// UnmarshalJSON unmarshal Network Selection Annotation. This also supports
+// the deprecated "interfaceRequest" property.
+func (net *NetworkSelectionElement) UnmarshalJSON(bytes []byte) error {
```

Comment:
where you call this function?

Reply:
UnmarshalJSON() will be invoked in Unmarshal(NetworkSelectionElement).

See followings:
- https://golang.org/pkg/encoding/json/#RawMessage.UnmarshalJSON
- https://mariadesouza.com/2017/09/07/custom-unmarshal-json-in-golang/

Reply:
Thanks for your explanation @s1061123 . Though I'm not really sure if types/conf.go is the right place to write this function.

Reply:
I'll look into unit tests for this.

Reply:
@giovanism we may change the function to somewhere in future, but for now, it is okey.